<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lblOpacity">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>320</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="WidgetInfo" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>55</y>
     <width>422</width>
     <height>162</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
background-color: white;</string>
   </property>
   <widget class="QPushButton" name="si">
    <property name="geometry">
     <rect>
      <x>211</x>
      <y>111</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 500 16px &quot;Poppins&quot;;
border-radius: 0px;
background-color: #266676;
color: white;</string>
    </property>
    <property name="text">
     <string>Si</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelInfo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>331</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16px &quot;Poppins&quot;;
background-color: transparent;
border: none;</string>
    </property>
    <property name="text">
     <string>Â¡Ajustes aplicados exitosamente!</string>
    </property>
   </widget>
   <widget class="QLabel" name="IconInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="no">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>111</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 500 16px &quot;Poppins&quot;;
border-radius: 0px;
background-color: #1a759f;
color: white;</string>
    </property>
    <property name="text">
     <string>No</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
